Sources/Core/Public/GTMSessionFetcher/GTMSessionUploadFetcher.h

```diff
@@ -133,6 +133,13 @@ typedef void (^GTMSessionUploadFetcherCancellationHandler)(GTMSessionFetcher *_N
 @property(atomic, copy) NSString *uploadMIMEType;
 @property(atomic, readonly, assign) int64_t chunkSize;
 @property(atomic, readonly, assign) int64_t currentOffset;
+@property(atomic, assign) double uploadRetryFactor;
+@property(atomic, assign) NSTimeInterval maxUploadRetryInterval;
+@property(atomic, assign) NSTimeInterval minUploadRetryInterval;
+typedef void (^GTMSessionFetcherExponentialBackoffBlock)(void);
```

Comment:
Actually, rereading things - how is a developer supposed to use this backoff block?  i.e. - what would they need it for? The test is using it, but that usage doesn't seem like what it would be generally used for.

Reply:
If this is just for the test, can we use the test hook or maybe the retry notifications in some way to hook in and cancel the fetcher instead?